doc/source/gallery/plotting_with_folium.ipynb

```diff
@@ -93,41 +75,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Stamen Terrain\n",
```

Comment:
I find the current overview of tiles quite helpful, do you think there's a need to remove them?